docs/data-sources/project_policy.md

```diff
@@ -0,0 +1,34 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "env0_project_policy Data Source - terraform-provider-env0"
+subcategory: ""
+description: |-
+  
+---
+
+# env0_project_policy (Data Source)
+
+
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- **project_id** (String) id of the project
+
+### Read-Only
+
+- **disable_destroy_environments** (Boolean) disable destroy environments
+- **id** (String) id of the policy
```

Comment:
Can we make this the first item in this section?

Reply:
I can change it manually, but the next time we run `generateDocs` it will change it. Seems like it arranges it alphabetically. Couldn't find any flag/config in the plugin [repo](https://github.com/hashicorp/terraform-plugin-docs)

Reply:
So forget about it, it's not really important.